plugin.go

```diff
@@ -169,6 +169,16 @@ func CopyTfEnv() {
 	}
 }
 
+func credsSet() bool {
```

Comment:
The way I'm reading it if **either** `AWS_ACCESS_KEY_ID` *or* `AWS_SECRET_ACCESS_KEY` is set it will return `true`.  But I think it should only return `true` if **both** are set, correct?

I think we should add a test for this function to help ensure it behaves as expected.

Reply:
@jmccann You are right! I changed the logic to return true only if all tokens are set. I also added a test for credsSet function as you suggested

Reply:
I suggest you to look the changes per commit

Reply:
Thanks!  Thanks! I'll try to find some time for a hopefully final review.  😄

Reply:
If you find it too hard to review it, maybe I could help adding more documentation into the code or add more info about how to use the feature on README, please let me know!

Reply:
This looks better now!